File: ARCADE/The Core/Level 4_4: lineup.cs
    using System;
     
    public class Test
    {
    	public static int lineUp(string commands){
            int result = 0, count = 0;
            for (int i = 0; i < commands.Length; i++){
                if (commands[i] == 'L' || commands[i] == 'R') count++;
                if (commands[i] == 'A') count += 0;
                if (count % 2 == 0) result++;
            }
            return result;
        }
    	public static void Main()
    	{
    		string commands = "LLARL";
    		Console.Write(lineUp(commands));
    	}
    }

File: ARCADE/The Core/Level 4_4: lineup.pas
program commands;

function lineUp(commands: string): integer;
var rs, i, count : integer;
begin
    rs := 0; count := 0;
    for i := 1 to length(commands) do
    begin
        if((commands[i] = 'L') or (commands[i] = 'R')) then inc(count);
        if(commands[i] = 'A') then inc(count, 0);
        if(count mod 2 = 0) then inc(rs);
    end;
    
    lineUp := rs;
end;

var command : string;
begin
	command := 'LLARL';
	writeln(lineUp(command));
end.
